lib/internal/http2/core.js

```diff
@@ -495,7 +498,7 @@ function onStreamClose(code) {
   // Defer destroy we actually emit end.
   if (!stream.readable || code !== NGHTTP2_NO_ERROR) {
     // If errored or ended, we can destroy immediately.
-    stream[kMaybeDestroy](code);
+    stream.destroy();
```

Comment:
Shouldn't this pass `code` as `err` into `destroy(err)`? I believe that's what `kDestroyMaybe` did.

Reply:
Pehaps you confused it with `sesion.kMaybeDestroy`, the `stream.kMaybeDestroy` is
https://github.com/nodejs/node/blob/024b6588313a0e32595317936ee279dec7a5a756/lib/internal/http2/core.js#L2131-L2160